fud/fud/stages/xilinx/execution.py

```diff
@@ -33,26 +35,15 @@ def _define_steps(self, input, builder, config):
                 f"is not. This will generate a WDB file but then immediately "
                 f"delete it. Consider adding `-s {self.name}.save_temps true`."
             )
-        
-        @builder.step()
-        def import_libs():
-            """Import optional libraries"""
-            try:
-                import pyopencl as cl  # type: ignore
-
-                self.cl = cl
-            except ImportError:
-                raise errors.RemoteLibsNotInstalled
```

Comment:
Wasn't sure about this, should the pynq import from `pyproject.toml` go here instead?

Reply:
The reason this is inside a `try`-`catch` inside a function is because the `fpga` dependencies are not installed by default. The program only throws an error if you try to run the `xilinx` stage without installing the `fpga` dependencies in the `flit` configuration.

I would recommend moving the
```
from fud.stages.xilinx import fud_pynq_script
```
and using a `self.pynq_script` field to access it instead of importing it on the top which will cause errors for default `fud` installations